### ISSUE TITLE
Demo doesn't work.

### ISSUE THREAD

**Issue by angeldollface** [angeldollface/lume-dark-material#2]
@kwaa The demo for your `mod.ts` doesn't work. Does it work for you? #1

**Comment by angeldollface** [comment]
@oscarotero This is the command output if I run `deno task serve` in the demo directory:

```text
Task serve deno task lume -s
Task lume echo "import 'lume/cli.ts'" | deno run --unstable -A - "-s"
Loading config file file:///Users/alex/Desktop/lume-dark-material/demo/_config.ts



Error: Error processing page
- page: /css/index.scss (remote: https://raw.githubusercontent.com/angeldollface/lume-dark-material/main/src/css/index.scss)
- processor: sass
    at (https://deno.land/x/lume@v1.17.5/core/processors.ts:63:21)
    at eventLoopTick (ext:core/01_core.js:189:13)
    at async concurrent (https://deno.land/x/lume@v1.17.5/core/utils.ts:160:3)
    at async Processors.run (https://deno.land/x/lume@v1.17.5/core/processors.ts:53:9)
    at async Site.#buildPages (https://deno.land/x/lume@v1.17.5/core/site.ts:663:5)
    at async Site.build (https://deno.land/x/lume@v1.17.5/core/site.ts:529:9)
    at async build (https://deno.land/x/lume@v1.17.5/cli/build.ts:41:3)
    at async Command.execute (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1794:7)
    at async Command.parseCommand (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1639:14)

Caused by Error: expected "{".
  ╷
1 │ 404: Not Found
  │               ^
  ╵
  css/index.scss 1:15  root stylesheet
    at Object.wrapException (file:///$HOME/npm/registry.npmjs.org/sass/1.62.1/sass.dart.js:1237:17)
    at SpanScanner.error$3$length$position 
```

**Comment by kwaa** [comment]
I didn't test it, which is a bit strange.

**Comment by angeldollface** [comment]
@kwaa Do you want to debug this? Because then I'll assign you.

**Comment by oscarotero** [comment]
The issue here: https://github.com/angeldollface/lume-dark-material/blob/main/mod.ts#L26

The remote files are registered in the `./src` directory.

**Comment by angeldollface** [comment]
@oscarotero @kwaa Thank you. Going to re-run the tests now.

**Comment by angeldollface** [comment]
Sitrep: @oscarotero @kwaa Even though we changed the path, it still fails. Running `deno task serve` still yields the same error.

**Comment by oscarotero** [comment]
@angeldollface The file passed to `import.meta.resolve()` must to start with `./`.

**Comment by kwaa** [comment]
> @angeldollface The file passed to `import.meta.resolve()` must start with `./`.

I kept it `./` prefix

@angeldollface Can you add a github action to test the demo?

**Comment by angeldollface** [comment]
So..the final one fails with a `x`not prefixed error. What is going on???

**Comment by kwaa** [comment]
I no longer get the prefix error locally, instead I get the same sass error as https://github.com/angeldollface/lume-dark-material/issues/2#issuecomment-1585645280.

```bash
Error: Error processing page
- page: /css/index.scss (remote: https://raw.githubusercontent.com/angeldollface/lume-dark-material/main/css/index.scss)
- processor: sass
    at (https://deno.land/x/lume@v1.17.5/core/processors.ts:63:21)
    at eventLoopTick (ext:core/01_core.js:182:11)
    at async concurrent (https://deno.land/x/lume@v1.17.5/core/utils.ts:160:3)
    at async Processors.run (https://deno.land/x/lume@v1.17.5/core/processors.ts:53:9)
    at async Site.#buildPages (https://deno.land/x/lume@v1.17.5/core/site.ts:663:5)
    at async Site.build (https://deno.land/x/lume@v1.17.5/core/site.ts:529:9)
    at async build (https://deno.land/x/lume@v1.17.5/cli/build.ts:41:3)
    at async Command.execute (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1794:7)
    at async Command.parseCommand (https://deno.land/x/cliffy@v0.25.7/command/command.ts:1639:14)

Caused by Error: expected "{".
  ╷
1 │ 404: Not Found
  │               ^
  ╵
  _includes/sass/_responsive.scss 1:15         @import
  _includes/sass/_lume-dark-material.scss 2:9  @import
  css/index.scss 15:9                          root stylesheet
```

I don't know much about sass, so I gave up.

**Comment by kwaa** [comment]
Well, it looks like `responsive.scss` needs to be renamed to `_responsive.scss`

**Comment by angeldollface** [comment]
@kwaa Initially, doing what you suggested, works! So the GitHub Action passes but initially the build failed on my machine. Steps I took to fix this:

- Removed the `deno` folder in the `$HOME/Library/Caches` folder.
- While the site would build, static assets wouldn't be loaded. This was fixed by leaving the `baseurl` field in `_data/site.json` empty.
- I'm not sure how important this is, but I'll mention it anyway: I also updated the routes in `_data/navigation.json` to exclude any routes to content that doesn't exist.

I should probably also remove the `*.markdown` files from the `mod.ts` because users of this theme need to write their own content.
